specification/chaos/resource-manager/Microsoft.Chaos/preview/2022-07-01-preview/types/common.json

```diff
@@ -0,0 +1,191 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Common Types",
+    "version": "2022-07-01-preview"
+  },
+  "paths": {},
+  "definitions": {
+    "resourceIdentity": {
+      "type": "object",
+      "description": "The managed identity of a resource.",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "String of the resource identity type.",
+          "enum": [
+            "None",
+            "SystemAssigned"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceIdentityType",
+            "modelAsString": false
+          }
+        },
+        "principalId": {
+          "type": "string",
+          "description": "GUID that represents the principal ID of this resource identity.",
+          "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+          "readOnly": true
+        },
+        "tenantId": {
+          "type": "string",
+          "description": "GUID that represents the tenant ID of this resource identity.",
+          "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+          "readOnly": true
+        }
+      },
+      "required": [
+        "type"
+      ],
+      "additionalProperties": false
+    },
+    "runtimeProperties": {
+      "type": "object",
+      "description": "The runtime properties of a resource.",
+      "properties": {
+        "kind": {
+          "type": "string",
+          "description": "String of the kind of the resource's action type (continuous or discrete).",
+          "readOnly": true
+        }
+      }
+    },
```

Comment:
Can this be in the `types/capabilityTypes.json` file? This definition isn't common across types.